src/languages/es.ts

```diff
@@ -1542,7 +1542,7 @@ export default {
         completed: 'Completada',
         messages: {
             completed: 'tarea completada',
-            canceled: 'tarea cancelada',
+            canceled: 'tarea eliminado',
```

Comment:
@thienlnam  I don't know where you got this translation from (I checked slack and could not find it anywhere), but it is wrong. Please make sure you always get confirmation in #espanol-tranlsations

Reply:
Sending a PR to fix this